tests/extract/test_extract_job.sql

```diff
@@ -1,179 +1,168 @@
+LEFT JOIN
+`test_treeherder`.`option` AS `t13` ON `t13`.`id` = `t8`.`option_id`
+LEFT JOIN
+`test_treeherder`.`repository` AS `t14` ON `t14`.`id` = `t10`.`repository_id`
+UNION ALL
 
-        SELECT
-            `t1`.`id` AS `c0`,  NULL  AS `c1`,  NULL  AS `c2`,  NULL  AS `c3`,  NULL  AS `c4`,  NULL  AS `c5`,  NULL  AS `c6`,  NULL  AS `c7`,  NULL  AS `c8`,  NULL  AS `c9`,  NULL  AS `c10`,  NULL  AS `c11`,  NULL  AS `c12`,  NULL  AS `c13`,  NULL  AS `c14`,  NULL  AS `c15`,  NULL  AS `c16`,  NULL  AS `c17`,  NULL  AS `c18`,  NULL  AS `c19`,  NULL  AS `c20`,  NULL  AS `c21`,  NULL  AS `c22`,  NULL  AS `c23`,  NULL  AS `c24`,  NULL  AS `c25`,  NULL  AS `c26`,  NULL  AS `c27`,  NULL  AS `c28`,  NULL  AS `c29`,  NULL  AS `c30`,  NULL  AS `c31`,  NULL  AS `c32`,  NULL  AS `c33`,  NULL  AS `c34`,  NULL  AS `c35`,  NULL  AS `c36`,  NULL  AS `c37`,  NULL  AS `c38`,  NULL  AS `c39`,  NULL  AS `c40`,  NULL  AS `c41`,  NULL  AS `c42`,  NULL  AS `c43`,  NULL  AS `c44`,  NULL  AS `c45`,  NULL  AS `c46`,  NULL  AS `c47`,  NULL  AS `c48`,  NULL  AS `c49`,  NULL  AS `c50`,  NULL  AS `c51`,  NULL  AS `c52`,  NULL  AS `c53`,  NULL  AS `c54`,  NULL  AS `c55`,  NULL  AS `c56`,  NULL  AS `c57`,  NULL  AS `c58`,  NULL  AS `c59`,  NULL  AS `c60`,  NULL  AS `c61`,  NULL  AS `c62`,  NULL  AS `c63`,  NULL  AS `c64`,  NULL  AS `c65`,  NULL  AS `c66`, `t2`.`id` AS `c67`, `t2`.`title` AS `c68`, `t2`.`value` AS `c69`, `t2`.`url` AS `c70`, `t2`.`job_id` AS `c71`,  NULL  AS `c72`,  NULL  AS `c73`,  NULL  AS `c74`,  NULL  AS `c75`,  NULL  AS `c76`,  NULL  AS `c77`,  NULL  AS `c78`,  NULL  AS `c79`,  NULL  AS `c80`,  NULL  AS `c81`,  NULL  AS `c82`,  NULL  AS `c83`,  NULL  AS `c84`,  NULL  AS `c85`,  NULL  AS `c86`,  NULL  AS `c87`,  NULL  AS `c88`,  NULL  AS `c89`,  NULL  AS `c90`,  NULL  AS `c91`,  NULL  AS `c92`,  NULL  AS `c93`,  NULL  AS `c94`,  NULL  AS `c95`,  NULL  AS `c96`,  NULL  AS `c97`,  NULL  AS `c98`,  NULL  AS `c99`,  NULL  AS `c100`,  NULL  AS `c101`,  NULL  AS `c102`,  NULL  AS `c103`,  NULL  AS `c104`,  NULL  AS `c105`,  NULL  AS `c106`,  NULL  AS `c107`,  NULL  AS `c108`,  NULL  AS `c109`,  NULL  AS `c110`,  NULL  AS `c111`,  NULL  AS `c112`,  NULL  AS `c113`,  NULL  AS `c114`,  NULL  AS `c115`,  NULL  AS `c116`,  NULL  AS `c117`,  NULL  AS `c118`,  NULL  AS `c119`,  NULL  AS `c120`,  NULL  AS `c121`,  NULL  AS `c122`,  NULL  AS `c123`,  NULL  AS `c124`,  NULL  AS `c125`,  NULL  AS `c126`,  NULL  AS `c127`,  NULL  AS `c128`,  NULL  AS `c129`,  NULL  AS `c130`,  NULL  AS `c131`,  NULL  AS `c132`,  NULL  AS `c133`,  NULL  AS `c134`,  NULL  AS `c135`,  NULL  AS `c136`,  NULL  AS `c137`,  NULL  AS `c138`,  NULL  AS `c139`,  NULL  AS `c140`,  NULL  AS `c141`,  NULL  AS `c142`,  NULL  AS `c143`
-        FROM
-            (SELECT 0) AS `t0`
-                LEFT JOIN
-            `test_treeherder`.`job` AS `t1` ON `t1`.`id` = `t0`.`id`
-                JOIN
-            `test_treeherder`.`job_detail` AS `t2` ON `t2`.`job_id` = `t1`.`id`
-        UNION ALL
```

Comment:
It is unfortunate that I left a formatted version of the SQL in the code, but it looks good:  The `job_detail` block has been removed.